### DISCUSSION TITLE
Naming Convention

### DISCUSSION THREAD

**Discussion by BrodieBrown1998** [gh44922023/Tafe230221#5] [Q&A] [answered]
Do you think the naming convention used was okay?
![Naming Convention](https://github.com/gh44922023/Tafe230221/assets/141377278/e8761187-bd5d-47b0-a9c5-741045cfd795)

**Comment by jeffrey-tafe** [comment, marked as answer]
It's unconventional but I don't see any issues for this project. I will use this format.

**Comment by jeffrey-tafe** [comment]
@gh44922023 Are you cool with this?

**Reply by gh44922023** [reply]
I too see no issues with this in this workspace.
